.github/workflows/ci.yml

```diff
@@ -122,21 +131,26 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - uses: Swatinem/rust-cache@v2
+
       - uses: "./.github/actions/init"
         with:
           git-user: ${{ env.GITHUB_ACTOR }}
 
+      - name: Install cargo-nextest
+        uses: taiki-e/install-action@v2
+        with:
+          tool: cargo-nextest
+
       - name: Run unit tests (no default features)
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: cargo test --lib --bins --no-default-features
+        run: cargo nextest run --lib --bins --no-default-features
```

Comment:
https://corrode.dev/blog/tips-for-faster-rust-compile-times/#use-cargo-nextest-instead-of-cargo-test